db/migrate/20200318135632_rename_report_templates.rb

```diff
@@ -0,0 +1,30 @@
+class RenameReportTemplates < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Ansible Inventory" => "Ansible - Ansible Inventory",
+    "Applicable errata" => "Host - Applicable Errata",
+    "Applied Errata" => "Host - Applied Errata",
+    "Entitlements" => "Subscription - Entitlement Report",
+    "Host statuses" => "Host - Statuses",
+    "Registered hosts" => "Host - Registered Content Hosts",
+    "Registered users" => "User - Registered Users",
+    "Subscriptions" => "Subscription - General Report",
+  }
+
+  class FakeReportTemplate < ApplicationRecord
+    self.table_name = 'templates'
+  end
+
+  def up
+    TEMPLATE_NAMES.each do |from, to|
+      FakeReportTemplate.find_by_name(to).delete
```

Comment:
Maybe something like:
```suggestion
      FakeReportTemplate.where(:name => to).update_all(:name => "#{to} Backup")
```

Reply:
That does not solve the problem, you have the very same brother. This time with "Backup" suffix :-)

Ok I will do something similar but with unique random characters.